tests/org.eclipse.e4.ui.tests/src/org/eclipse/e4/ui/workbench/renderers/swt/StackRendererTest.java

```diff
@@ -540,7 +541,9 @@ public void testOnboardingIsHiddenWhenEditorOpened() {
 		partStack.getChildren().add(part1);
 		partStack.setSelectedElement(part1);
 
-		tabFolder.notifyListeners(SWT.Paint, new Event());
+		Event event = new Event();
+		event.gc = new GC(tabFolder);
+		tabFolder.notifyListeners(SWT.Paint, event);
```

Comment:
```suggestion
		tabFolder.redraw();
```
Why do we need a synthetic event? The vent is also missing the widget / source and so on... also GC has to be disposed afterwards-

Reply:
@marcushoepfner: Can you answer this question? Do you think the proposed change from @laeubi would also be ok?

Reply:
Not sure. One has to test it. If the test is green, it works.
And the "synthetic" event is used because Eclipse is lacking nice test doubles. And it seems that this was just about notification, widget was not needed for the unit test.

Reply:
I checked it locally and for me it seems, the notify statement can be removed completely. 
I remember that we had a paint listener for the tab folder. after a lot of fixes and refactorings that seem not to be the case anymore.
so IMHO `tabFolder.notifyListeners(SWT.Paint, event);` can be removed. can you please change this @BeckerWdf ?

Reply:
> I checked it locally and for me it seems that the notify statement can be removed completely. I remember that we had a paint listener for the tab folder. after a lot of fixes and refactorings that seem not to be the case anymore. so IMHO `tabFolder.notifyListeners(SWT.Paint, event);` can be removed. can you please change this @BeckerWdf ?

Sure. Will do.